sources/packages/web/src/components/common/modals/OkCancelModal.vue

```diff
@@ -0,0 +1,64 @@
+<template>
```

Comment:
IMO the name of the modal can be aligned with it's purpose. e.g ConsentModal or ConfirmModal something like that.

Reply:
Modal renamed to "ConfirmModal'.